daemon/src/model/cfd.rs

```diff
@@ -261,6 +261,10 @@ pub enum CfdEvent {
     CollaborativeSettlementProposed {
         proposal: SettlementProposal,
     },
+    CollaborativeSettlementProposalReceived {
```

Comment:
Let's do this as discussed with using a `CollaborativeSettlementStarted` for both, similar to what we prepared for rollover.